galsim/phase_psf.py

```diff
@@ -1068,15 +1080,18 @@ def __str__(self):
 
     def __repr__(self):
         outstr = ("galsim.PhaseScreenPSF(%r, lam=%r, exptime=%r, flux=%r, aper=%r, theta=%r, " +
-                  "scale_unit=%r, interpolant=%r, gsparams=%r)")
+                  "interpolant=%r, scale_unit=%r, gsparams=%r)")
```

Comment:
> Does that mean there's a missing unit test? `do_pickle(phasescreen_psf)`

This took a bit of tinkering to figure out.  We did have some `do_pickle(phasescreen_psf)`s, but none of them triggered an error for two reasons:

1) The existing ones were all for atmospheric phase screens that have an `rng` class member and so the eval(repr()) test gets skipped since we don't expect the BaseDeviates to pass an eval(repr()) roundtrip.
2) When I made a `do_pickle` on an `OpticalScreen` `do_pickle` test, the try/except block designed to catch unevaluatable reprs triggered, skipping the test.

So I rewrote the eval/repr roundtrip test, and got this error to trigger.  Doing that also brought up a bunch of other errors, though, so I went through and fixed all of them.